### ISSUE TITLE
Tensorflow Lite Support for Windows

### ISSUE THREAD

**Issue by apollo-time** [tensorflow/tensorflow#14607]
Are you planning to support Tensorflow Lite on Windows? Specifically Windows 32bit.

**Comment by aselle** [comment]
What kind of support are you interested in? Inference or tooling for preparing models?

**Comment by apollo-time** [comment]
I need inference on the 32bit windows with pretraind model,, because there are a lot of 32bit windows devices yet and our product need support all windows devices.
How can I run the pretraind tensorflow model on the 32bit windows?
Thanks.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by aselle** [comment]
See also
#16374

**Comment by aselle** [comment]
@apollo-time, you can probably make a very simple build that supports windows. There is also a pending PR that adds support 
#16490
but it has not been merged into mainline yet.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by TsinghuaTop** [comment]
@apollo-time you can see https://github.com/freeors/aismart/tree/master/aismart-src/aismart/external/tensorflow ,this preject had build tflite on windows , docs in https://zhuanlan.zhihu.com/p/32190069

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by srhodes1-lenovo** [comment]
Hey guys, any word on if/when this is going to happen?  This is something we are really looking for (not the part about 32 bit specifically, just a TF-lite for Windows build in general).

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!

**Comment by adis300** [comment]
This is an issue. Lite should can support UWP/WPF/Form apps and potentially MacOS native apps.

**Comment by srhodes1-lenovo** [comment]
It's still an issue here as well. We'd really like to have a Windows build of TF-Lite.  It's not a hard blocker right this minute, but it's something we could definitely use.

**Comment by weizhuang-93** [comment]
uwp build on 32 bit system is really an issue

**Comment by fzyzcjy** [comment]
+1 Still an issue, want to use the same code package for both Windows and Android

**Comment by synap5e** [comment]
Having to include the full tensorflow package to preform inference in a windows app is not ideal for me, so I'm looking forward to this issue being resolved.

**Comment by WesleyKhoiVo** [comment]
microsoft/vcpkg#14394

**Comment by WesleyKhoiVo** [comment]
https://github.com/microsoft/vcpkg/pull/14394

**Comment by WesleyKhoiVo** [comment]
[](https://github.com/microsoft/vcpkg/pull/14394)

**Comment by WesleyKhoiVo** [comment]
[](https://github.com/microsoft/vcpkg/pull/14394)
